frontend/src/components/organisms/forms/sponsorForm.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { ModalDrawer } from "components/organisms";
+
+const Component = (props: { event?: any }) => {
+  const onSubmit = () => {
+    // TODO: send data to server
+    // if (Object.keys(errors).length === 0)
+    // console.log(data)
+    //sponsorApi
+    //.createSponsor(data)
+    //.then(() => onSubmitSuccess())
+    //.catch((error) => onSubmitFailed(utils.parseErrors(error)));
+    //}
+  };
+
+  return (
+    <ModalDrawer
+      title="Sponsor"
+      opener={{
+        title: `Sponsor Offer`,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9091638)
```suggestion
        title: "Sponsor Offer",
```